src/DssSpell.t.sol

```diff
@@ -101,48 +89,68 @@ contract DssSpellTest is DssSpellTestBase {
         }
     }
 
-    function testCollateralIntegrations() private { // make public to use
+    function testCollateralIntegrations() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // checkIlkIntegration(
-        //     "RETH-A",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
-        //     addr.addr("PIP_RETH"),
-        //     true, /* _isOSM */
-        //     true, /* _checkLiquidations */
-        //     false /* _transferFee */
-        // );
+        checkIlkIntegration(
+            "RETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
+            addr.addr("PIP_RETH"),
+            true, /* _isOSM */
+            true, /* _checkLiquidations */
+            false /* _transferFee */
+        );
     }
 
-    function testIlkClipper() private { // make public to use
+    function testIlkClipper() public { // make private to disable
         vote(address(spell));
         scheduleWaitAndCast(address(spell));
         assertTrue(spell.done());
 
-        // // Insert new ilk clipper tests here
-        // checkIlkClipper(
-        //     "TOKEN-X",
-        //     GemJoinAbstract(addr.addr("MCD_JOIN_TOKEN_X")),
-        //     ClipAbstract(addr.addr("MCD_CLIP_TOKEN_X")),
-        //     addr.addr("MCD_CLIP_CALC_TOKEN_X"),
-        //     OsmAbstract(addr.addr("PIP_TOKEN")),
-        //     20_000 * WAD
-        // );
+        checkIlkClipper(
+            "RETH-A",
+            GemJoinAbstract(addr.addr("MCD_JOIN_RETH_A")),
+            ClipAbstract(addr.addr("MCD_CLIP_RETH_A")),
+            addr.addr("MCD_CLIP_CALC_RETH_A"),
+            OsmAbstract(addr.addr("PIP_RETH")),
+            2400 * WAD
```

Comment:
This should be lowered as it is now over the `line` and causes the test to fail, 500 should be fine.
Please also change it in goerli as well (so it gets archived as well).